_inc/lib/core-api/wpcom-endpoints/publicize-connection-test-results.php

```diff
@@ -105,6 +105,10 @@ public function get_items( $request ) {
 			}
 		}
 
+		if ( 'linkedin' === $item['id'] && 'must_reauth' === $test_result['connectionTestPassed'] ) {
```

Comment:
@eliorivero Is this a bug? It looks like the if statement should be inside the foreach loop above it. That's how it is on WP.com.